tests/testthat/helper.R

```diff
@@ -16,17 +16,13 @@ if (!is.null(session_temp_proj)) {
   ))
 }
 
-## putting `pattern` in the package or project name is part of our strategy for
-## suspending the nested project check during testing
-pattern <- "aaa"
```

Comment:
TBH honest I'm not sure why I ever made the magic string part of the default project name.

Probably because I was very frustrated with problems around nested projects and testing? If this ever happens to you @Hadley (catastrophic test failure related to nested projects), remember we have some code elsewhere in this `helper.R` file that can be run to figure out why session temp dir suddenly looks like a project. Then we can delete the offending file. This is something RStudio does in the background every now and then and I've never been able to figure out why.